scripts/test/support.py

```diff
@@ -152,6 +157,8 @@ def to_end(j):
             ignoring_quoted = True
             continue
         if chunk.startswith('(module'):
+            if MODULE_DEFINITION_OR_INSTANCE.match(chunk):
+                continue
             ignoring_quoted = False
```

Comment:
We should probably update the name of `ignoring_quoted`. Maybe `ignoring_module`?